ledger-core/virtual/integration/benchs/semi_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package benchs
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	walletproxy "github.com/insolar/assured-ledger/ledger-core/application/builtin/proxy/testwallet"
+	"github.com/insolar/assured-ledger/ledger-core/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/contract"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/runner/executor/common/foundation"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/synckit"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/integration/utils"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils"
+)
+
+func BenchmarkOnWallets(b *testing.B) {
+	server, ctx := utils.NewServer(nil, b)
+	defer server.Stop()
+
+	resultSignal := make(synckit.ClosableSignalChannel, 1)
+
+	wallets := make([]reference.Global, 0, 1000)
+
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, b, server)
+	typedChecker.VCallResult.Set(func(result *payload.VCallResult) bool {
+		if result.CallType == payload.CTConstructor {
+			var (
+				err             error
+				ref             reference.Global
+				contractCallErr *foundation.Error
+			)
+			err = foundation.UnmarshalMethodResultSimplified(result.ReturnArguments, &ref, &contractCallErr)
+			require.NoError(b, err)
+			require.Nil(b, contractCallErr)
+			wallets = append(wallets, ref)
+		}
+		resultSignal <- struct{}{}
+		return false
+	})
+
+	pl := payload.VCallRequest{
+		CallType:       payload.CTConstructor,
+		CallFlags:      payload.BuildCallFlags(contract.CallTolerable, contract.CallDirty),
+		Caller:         server.GlobalCaller(),
+		Callee:         walletproxy.GetClass(),
+		CallSiteMethod: "New",
+		Arguments:      insolar.MustSerialize([]interface{}{}),
+	}
+
+	for i := 0; i < 1000; i++ {
```

Comment:
I prefer to use constant and comment for this purpose. Please no magic numbers)

Reply:
the test is run on 1000 wallets. I don't use this 1000 anymore. In what scope do you want to see the constant?

Reply:
Comments will be enough here. In my vision, we always should use constant+comment instead of value in code.

Reply:
added comment. I don't understand what exactly you want to achieve with constant here.

Reply:
agreed with @sexyhamster - placing constant here gives nothing but polluting code with unnecessary content